packages/interactive/lib/src/workspace_isolate.dart

```diff
@@ -70,5 +86,22 @@ Future<Isolate> _spawnUriWithErrorHandling(Uri uri) async {
     null,
     onError: errorPort.sendPort,
     onExit: exitPort.sendPort,
+    packageConfig: packageConfigUri,
+  );
+}
+
+Future<void> _runPubGet({required String workingDirectory}) async {
+  final process = await Process.start(
+    '/bin/sh',
```

Comment:
this line won't work on windows

Reply:
@JordyScript Yes, I've noticed later when it ran in the CI. I've already updated it, although it looks there is an unrelated issue going on with Windows when running the tests, which also happens in the master branch #90